lib/validator/validator.go

```diff
@@ -129,43 +138,43 @@ func validateLabels(v *validator.Validate, topStruct reflect.Value, currentStruc
 
 func validateInterface(v *validator.Validate, topStruct reflect.Value, currentStructOrField reflect.Value, field reflect.Value, fieldType reflect.Type, fieldKind reflect.Kind, param string) bool {
 	b := []byte(field.String())
-	log.Infof("Validate interface: %s", b)
+	log.Debugf("Validate interface: %s", b)
 	return nameRegex.Match(b)
```

Comment:
For linux interface names, we use a different regex in felix, limiting the length and characters: `^[a-zA-Z0-9_-]{1,15}$`

Not sure if we should allow `.` and `:` in there too; probably needs some research.